src/defaults.json

```diff
@@ -1,4 +1,9 @@
 {
   "spacingAliases": {},
+  "displayAliases": {
+    "small": "-599",
```

Comment:
It's nice how terse this syntax is, but it's a bit weird to read. Would you still prefer this to something like:
`{ small: { minWidth: 599 }, medium: { minWidth: 600, maxWidth: 899 }, ... }`

This would also be a little bit closer to how media queries are typically written (with the `minWidth/maxWidth` keywords)

Reply:
I guess it's a trade off between complexity of the API and flexibility but I like it because this approach enables us to add more formats later on (`e.g. { large: { orientation: 'landscape' } }`